README.md

```diff
@@ -45,9 +45,10 @@ Other requirements are R, [htslib](https://github.com/samtools/htslib) and [vt](
 #### Results
 
 The main output files are:
-* [clinvar.tsv.gz](clinvar.tsv.gz)  
-* [clinvar.vcf.gz](clinvar.vcf)  
-* [clinvar_with_exac.tsv.gz](clinvar_with_exac.tsv.gz)  
+* [clinvar_record.tsv.gz](clinvar_record.tsv.gz): variant-condition specific record
```

Comment:
to avoid people being confused about the difference between clinvar_record.tsv and clinvar.tsv, how about we call it `clinvar_allele_trait_pairs.tsv.gz` and the other file `clinvar_alleles.txt.gz` instead of `clinvar.tsv.gz` (and then `clinvar_alleles_with_exac.txt.gz`)